lib/src/analyzer_helpers.dart

```diff
@@ -180,8 +180,6 @@ String _copyFieldDeclaration(FieldDeclaration decl, String initializer) {
 }
 
 String _copyGetterDeclaration(MethodDeclaration decl, String body) {
-  bool isAsync =
-      (decl.returnType?.type?.name ?? decl.returnType?.toString()) == 'Future';
```

Comment:
`DartType.name` was deprecated, so this had to be changed. The `keyword` solution below should be more robust, anyways